test/E2ETests/Automation.CLI/CLITestBase.cs

```diff
@@ -75,6 +75,10 @@ public void InvokeVsTestForExecution(string[] sources, string runSettings = "",
             // this step of Initializing extensions should not be required after this issue: https://github.com/Microsoft/vstest/issues/236 is fixed
             vsTestConsoleWrapper.InitializeExtensions(Directory.GetFiles(this.GetTestAdapterPath(), "*TestAdapter.dll"));
             vsTestConsoleWrapper.RunTests(sources, runSettingXml, new TestPlatformOptions { TestCaseFilter = testCaseFilter }, this.runEventsHandler);
+            if (this.runEventsHandler.Errors.Any())
+            {
+                throw new Exception($"Run failed with {this.runEventsHandler.Errors.Count} errors:{Environment.NewLine}{string.Join(Environment.NewLine, this.runEventsHandler.Errors)}");
```

Comment:
Should this be a localized exception ?

Reply:
This is our test code, it's fine. We won't spend cash localizing those errors 🙂